internal/usecase/client/pendo/pendo_client_dump.go

```diff
@@ -0,0 +1,50 @@
+package pendo
+
+import (
+	"context"
+	"log/slog"
+
+	app_context "github.com/podengo-project/idmsvc-backend/internal/infrastructure/context"
+	"github.com/podengo-project/idmsvc-backend/internal/interface/client/pendo"
+)
+
+type pendoClientDump struct{}
+
+// NewClientDump
+func NewClientDump() pendo.Pendo {
+	return newClientDump()
+}
+
+// SetMetadata launch a request to pendo service to register some metric
+func (c *pendoClientDump) SetMetadata(ctx context.Context, kind pendo.Kind, group pendo.Group, metrics pendo.SetMetadataRequest) (*pendo.SetMetadataResponse, error) {
+	logger := app_context.LogFromCtx(ctx)
+	logger.Debug("pendo SetMetadata called",
+		slog.String("kind", string(kind)),
+		slog.String("group", string(group)),
+	)
+	metaDataResponse := &pendo.SetMetadataResponse{}
+	logger.Debug("pendo SetMetadata response",
+		slog.Int64("total", metaDataResponse.Total),
+		slog.Int64("failed", metaDataResponse.Failed),
+		slog.Int64("updated", metaDataResponse.Updated),
+		slog.Any("missing", metaDataResponse.Missing),
+		slog.String("kind", string(metaDataResponse.Kind)),
+	)
+	return metaDataResponse, nil
+}
+
+// SendTrackEvent
+// See: https://engageapi.pendo.io/?bash%23#e45be48e-e01f-4f0a-acaa-73ef6851c4ac
+func (c *pendoClientDump) SendTrackEvent(ctx context.Context, track *pendo.TrackRequest) error {
+	logger := app_context.LogFromCtx(ctx)
+	logger.Debug("pendo SendTrackEvent successful")
```

Comment:
The "successful" can be confusing as nothing was actually sent.  We can use the same as in the other method. E.g. `pendo SendTrackEvent called`.